includes/wsuwp-shortcode-fundselector.js

```diff
@@ -9,17 +9,29 @@ jQuery(document).ready(function($) {
 				$("#fundSearch").addClass("loading");
 				$.getJSON( wpData.request_url_base + 'idonate_fund', 
 					{
-						search : request.term
+						search : request.term,
+						per_page: 100,
```

Comment:
Since you are having the server do the sorting, you can reduce the amount of results to 11. The server will sort the results before sending them to you, so you reduce the amount of data sent back.
I tested this and got the same sorted results with 100 and with 11.